lib/cucumber/formatter/backtrace_filter.rb

```diff
@@ -13,13 +13,15 @@ module Formatter
       minitest
       test/unit
       .gem/ruby
-      lib/ruby/
       bin/bundle
     ]
 
+    @backtrace_filters << RbConfig::CONFIG['rubyarchdir']
```

Comment:
we should probably use a conditional to only push in the one relevant to the platform. A bit like the JRuby one below

Reply:
This was my initial idea, then I went with the `#compact`. But I can modify the PR if you prefer the conditional.

Reply:
Ah I missed that. Urm My preference would be not to mix and match, so I think it would be better using a conditional as that's what is done above.

If you look in `cucumber/core/platform.rb` you can add some constants there if you'd prefer.